public/services/session_load_service.ts

```diff
@@ -21,13 +21,13 @@ export class SessionLoadService {
     this.status$.next('loading');
     this.abortController = new AbortController();
     try {
-      return await this._http.get<ISession>(`${ASSISTANT_API.SESSION}/${sessionId}`, {
+      const payload = await this._http.get<ISession>(`${ASSISTANT_API.SESSION}/${sessionId}`, {
         signal: this.abortController.signal,
       });
+      this.status$.next('idle');
+      return payload;
     } catch (error) {
       this.status$.next({ status: 'error', error });
-    } finally {
-      this.status$.next('idle');
```

Comment:
The status won't be reset to idle when there is an error. Will this change affect other existing status$ consumers?

Reply:
There is only one consumer for session load service `status$`. The status$ should always be error when fail to load session in chat page.

Reply:
Yeah, error or idle both makes sense when there is an error. Not a big concern.